.setup/install_system.sh

```diff
@@ -619,30 +605,29 @@ if [ ${WORKER} == 1 ]; then
     echo  Running configure submitty in worker mode
     python3 ${SUBMITTY_REPOSITORY}/.setup/CONFIGURE_SUBMITTY.py --worker
 else
-    if [ ${VAGRANT} == 1 ]; then
-    # This should be set by setup_distro.sh for whatever distro we have, but
-    # in case it is not, default to our primary URL
-    if [ -z "${SUBMISSION_URL}" ]; then
-        SUBMISSION_URL='http://192.168.56.101'
-    fi
-    echo -e "/var/run/postgresql
-    ${DB_USER}
-    ${DATABASE_PASSWORD}
-    America/New_York
-    ${SUBMISSION_URL}
-
+        if [ ${VAGRANT} == 1 ]; then
```

Comment:
Why did the indentation of this line change?
(Or is this a tabs/spaces thing that github is making look worse?)

Overall the other indentation changes are probably an improvement (but are unrelated to emma & should not affect the functionality of the script, right?)

Reply:
Yeah, the indentation shouldn't have any effect on things. The indentation of the contained lines was at the same level as `if [ ${VAGRANT}...` which was a bit confusing and so I indented it, though included this line by accident.

Fixed it so that the indentation is as it should be, while also removing the indentation on all of the echoed lines to better indicate their final values (CONFIGURE_SUBMITTY.py still applies a `.strip()` to any input though).